app/policies/casa_case_policy.rb

```diff
@@ -68,24 +70,26 @@ def permitted_attributes
     end
   end
 
-  def show?
+  def same_org_supervisor_admin_or_assigned?
     is_in_same_org? && (
       is_supervisor_or_casa_admin? || is_volunteer_actively_assigned_to_case?
     )
   end
 
-  def new?
+  def same_org_supervisor_admin?
     is_in_same_org? && is_supervisor_or_casa_admin?
   end
 
   def index?
     user.casa_admin? || user.supervisor? || user.volunteer?
   end
 
-  alias_method :edit?, :show?
-  alias_method :update?, :show?
-  alias_method :create?, :new?
-  alias_method :destroy?, :new?
+  alias_method :show?, :same_org_supervisor_admin_or_assigned?
+  alias_method :edit?, :same_org_supervisor_admin_or_assigned?
+  alias_method :update?, :same_org_supervisor_admin_or_assigned?
+  alias_method :new?, :same_org_supervisor_admin?
+  alias_method :create?, :same_org_supervisor_admin?
+  alias_method :destroy?, :same_org_supervisor_admin?
```

Comment:
The default style for Pundit is to have methods named `show?`, `new?` etc. What about instead of having aliases, just having a private method that calls `same_org_supervisor_admin_or_assigned`? Like:

```
def new?
  same_org_supervisor_admin_or_assigned
end

def show?
  same_org_supervisor_admin_or_assigned
end

private

def same_org_supervisor_admin_or_assigned
  is_in_same_org? && is_supervisor_or_casa_admin?
end
```

It isn't really less code, but it sticks with the style that one expects when using Pundit.

Reply:
I think alias is the same as def in this case, is it not?